2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -9,3 +9,5 @@
 		# in both loops, if a match is found, cut those two characters out of the string and reform as a new string
 			# if this happens decrement strlen by 2
   # return strlen after the loops have finished
```

Comment:
Comment for line #3 - Should add that removing one these instances can create a new instance.

Reply:
Comment for Line 6-8 - It's unclear whether you're your looping multiple times or just performing operations inside the loop. Line 6 says you're looping, but then line 7 and 8 say there are more loops or are you just saying these are the checks to perform in the loop?

Reply:
my idea was something like, there's an outer loop that's looping through the string and then on each inner loop we loop through the whole string looking for matches. so three loops in total with two nested at the same level inside of the outer loop. not really sure if this will work though ^^;